app/Models/Multiplayer/Room.php

```diff
@@ -500,6 +525,15 @@ public function recentParticipants(): array
             ->all();
     }
 
+    public function save(array $options = [])
+    {
+        if ($this->exists && $this->isDirty('ends_at')) {
```

Comment:
Wouldn't it make more sense for this column to be called `ended_at` similar to `scores` table, and have it set when the score is set, rather than when the room is saved?

Reply:
It's duplicating the room's `ends_at` time and ordered to match the display of a room's end time, not when the user played
![image](https://github.com/ppy/osu-web/assets/1694544/d67b773a-8a30-4cc4-baae-1128a469896a)

Reply:
I get that, but I'm proposing not doing that. I think it makes more sense to list based on the user's last play rather than the time the room was closed? Will make much more sense for the profile page usage.

Reply:
The display will need to be adjusted a bit since the part with the room state and closing time are meant to make sense together, and it would be weird if all the times started showing other of order.